src/MooLite/plugins/WhisperManager/DevConstants.ts

```diff
@@ -0,0 +1,565 @@
+import { ChatChannelTypeHrid } from "src/MooLite/core/chat/ChatChannelTypeHrid";
+import { ConversationMessage } from "src/MooLite/plugins/WhisperManager/ConversationMessage";
+
+export const contributorNames: string[] = ["LagunaE", "Sleutelkind"];
+export const betaTesterNames: string[] = ["JBQ", "knighterd", "Thessaloniki", "Houston"];
+export const helperNames: string[] = ["Champ", "ML", "Maxion", "hoador"];
+
+export const conversationData: Record<string, ConversationMessage[]> = {
```

Comment:
This Dev data can either be removed or rewritten into a test case

Reply:
I didn't see that any test framework was being used here. Do we want to include one?